.github/workflows/integration-tests.yml

```diff
@@ -55,13 +57,17 @@ jobs:
           - prove_empty_block
         zkvm: [sp1, risc0, pico, zisk, openvm]
         el: [reth, ethrex]
+        include:
+          - zkvm: openvm
+            threads: 2
```

Comment:
This is an old setup since OpenVM uses quite a lot of RAM per case, so going very far in parallelization makes the CI machine struggle. But since OpenVM emulation is very fast, doesn't hurt much CI duration.

For the rest, I configured 12 threads to avoid going with 16 which is the default for the CI machine. Mainly to give some stability to wall-clock time runs.